docs/src/content/docs/commands/PING.md

```diff
@@ -22,58 +16,84 @@ PING [message]
+   - `Error message`: `(error) ERR wrong number of arguments for 'ping' command`
+   - `Scenario`: If more than one argument is provided.
+
+   ```bash
+   127.0.0.1:7379> PING "Message 1" "Message 2"
+   (error) ERR wrong number of arguments for 'ping' command
+   ```
 
 ## Example Usage
 
-### Example 1: Pinging the server without a message
+### Basic Usage 
+
+`PING` DiceDB server without a message and the server echoes with `PONG`.
 
 ```bash
 127.0.0.1:7379> PING
 PONG
 ```
 
-### Example 2: Pinging the server with a message
+### Pinging the server with a message
+
+DiceDB server is pinged with `Hello, DiceDB!` and the server echoes with `Hello, DiceDB!`.
 
 ```bash
 127.0.0.1:7379>  PING "Hello, DiceDB!"
 "Hello, DiceDB!"
 ```
 
-## Behaviour
-
-When the `PING` command is fired:
+### Pinging the server with int message
 
-1. If no message is given, it sends back a `PONG`.
-2. If a message is provided, it sends back the same message in the response.
+DiceDB server is pinged with `1234` and the server echoes with `"1234"` coerced to string internally.
 
-This helps clients determine if the server is up and responsive. Essentially, it acts as a keep-alive or heartbeat mechanism for clients to validate their connection with the DiceDB server.
-
-## Error Handling
-
-### Potential Errors
-
-1. `Syntax Error`: If the syntax is incorrect, such as including unexpected additional parameters, an error will be raised:
+```bash
+127.0.0.1:7379>  PING 1234
+"1234"
+```
 
-   - `Error message`: `(error) ERR wrong number of arguments for 'ping' command`
-   - `Scenario`: If more than one argument is provided.
+### Pinging the server with list message
 
-   ```bash
-   > PING "Message 1" "Message 2"
-   (error) ERR wrong number of arguments for 'ping' command
-   ```
+DiceDB server is pinged with `[1234]` and the server echoes with `"[1234]"` coerced to string internally.
 
-2. `Data Type Error`: If the argument provided is not a string (for example, if a list or other data type is provided), a type error will be raised.
+```bash
+127.0.0.1:7379>  PING [1234]
+"[1234]"
+```
 
-   - `Error message`: This specific type of error handling is internal, and improper types are generally implicitly converted or rejected by the protocol, raising basic syntax errors or allowing them based on DiceDB internal type coercion rules.
+### Invalid Usage
 
-## Additional Notes
+DiceDB server is pinged with multiple parameters `"hello" "world"` and the server echoes with error message as `"(error) ERR wrong number of arguments for 'ping' command"`.
 
-- The `PING` command works similarly in both standalone and clustered DiceDB environments.
-- It is typically sent periodically by clients to ensure the connection is still active.
-- The `PING` command does not modify the data within the DiceDB server or affect any ongoing transactions.
+```bash
+127.0.0.1:7379>  PING "hello" "world"
+"(error) ERR wrong number of arguments for 'ping' command"
+```
```

Comment:
Isn't this the same as the errors section above? We can probably remove it.

Reply:
Removed the redundant section. Please review when you are free.
Thanks